### PR TITLE
Further snags

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Various small tweaks to address issues raised by Magda in issue queue

## QA

- Run code and check site
- Sanity check code

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ title: snapcraft
 email: anthony.dillon@canonical.com
 description: Introducing snaps and commands to install and remove them on Ubuntu 16.04 LTS.
 keywords: snappy, Ubuntu Core, Ubuntu 16.04 LTS, snaps, apps, debs, snap, docker, coreos, atomic, snapcraft, snapd
-baseurl: ""
+baseurl: "/snapcraft.io"
```
Comment (nottrobin) [inline _config.yml]: Can we not do this? we should prefer a config which works on live over one which works on GitHub pages. Please just make this change in your personal `gh-pages` branch.

File: _includes/header.html [hunk 2]
```diff
@@ -4,6 +4,11 @@
             <a href="#main-content">Jump to main content</a>
         </span>
         <div class="logo">
+            {% if jekyll.environment == "production" %}
```
Comment (nottrobin) [inline _includes/header.html]: Why is this necessary? What is it that ells jekyll it's on the production environment? I'm not aware of having set anything.

File: _includes/header.html [hunk 3]
```diff
@@ -4,6 +4,11 @@
...
+                <a href="{{site.baseurl}}/">
+            {% else %}
+                <a href="/">
+            {% endif %}
             <a href="/">
```
Comment (nottrobin) [inline _includes/header.html]: This seems like it's set above in the if block, and then set again here. Won't you end up with 2 `<a href="/">`?

File: index.html [hunk 4]
```diff
@@ -20,7 +20,7 @@ <h1 class="hero-row__heading">Snapcraft</h1>
         </div>
     </section>
 
-    <section class="row hero-row no-border">
+    <section class="row hero-row no-border" style="background:#f7f7f7">
```
Comment (nottrobin) [inline index.html]: Is it so late that we need inline styles? That's a real shame.

File: _includes/header.html (hunk 2 above)
Comment (barrymcgee) [inline _includes/header.html]: Github pages reports back to Jekyll as 'production'

Comment (nottrobin) [inline _includes/header.html]: Okay then, once again I don't think we should be making changes to the central codebase to accommodate `gh-pages`. As above, you can make these changes to your local `gh-pages` branch if you want, hopefully they'll merge okay with any new work.

File: index.html (hunk 4 above)
Comment (nottrobin) [inline index.html]: Conversation continued on IRC:

``` irc
barrymcgee: re: inline styling - prob not the time to have this discussion but when there's a requirement to make something in isolation an arbitrary style, the choice between adding three isolated lines of css in a css files, or one inline style, imo it's six of one, half a dozen or the other - neither tie in to any cohesive framework
robin: yes okay that's fair enough. Obviously we should be pushing back on specific styling like that, but I agree now probably isn't the time for that.
```

**Comment by nottrobin** [comment]
Code & QA now looks good. Thanks. Merging.
